OpenOrchestrator/orchestrator/popups/trigger_popup.py

```diff
@@ -49,7 +49,8 @@ def __init__(self, trigger_tab: TriggerTab, trigger_type: TriggerType, trigger:
             self.path_input = ui.input("Process Path").classes("w-full")
             self.git_check = ui.checkbox("Is path a Git Repo?")
             self.args_input = ui.input("Process Arguments").classes("w-full")
-            self.blocking_check = ui.checkbox("Is process blocking?")
+            checkbox_value = {'value': True} if not trigger else {}  # Make trigger blocking by default
+            self.blocking_check = ui.checkbox(text="Is process blocking?", **checkbox_value)
```

Comment:
You can just set the value to True. If a trigger is present the value is overwritten in line 98

```suggestion
            self.blocking_check = ui.checkbox(text="Is process blocking?", value=True)
```